elated-egrets/bangazon-internal-site-2.0
# Ls category

## DESCRIPTION
references [ticket](https://github.com/elated-egrets/bangazon-internal-site-2.0/issues/8) 
## How to pull down
Commands

```
git fetch --all
git checkout lsCategory
```

## Steps to test

1. Start virtual environment
1. delete your current database and migrations
1. `python manage.py makemigrations website && python manage.py migrate`
1. Make sure Django server is running
1. use db browser to add 1 or more categories to table website_categories
1. use [this webform](http://127.0.0.1:8000/sell) to create a few products of various categories
1. go to [/categories](http://127.0.0.1:8000/categories)
1. it should list all the categories, how many listings are in them, and **up to** three products from it


## What you changed

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
